### PR TITLE
The Congo Chronicles News Website

### PR DESCRIPTION
**Pull request by MayowaShoyinka**
The Congo Chronicles News Website
created a branch called tc-html-cleanup, renamed folders and linked them appropriately

### CODE REVIEW

**Review by MYKEL7 — COMMENTED** [review]
Nice job @MayowaShoyinka 
Pls also make the logo of each pages to be equal in size (Not too big). Also the social media icon should be the same for all pages, that can go to the footer.

File: homepage/homepage.html [hunk 1]
```diff
@@ -14,8 +14,25 @@ <H1>THE CONGO CHRONICLES</H1>
...
         </header>
+
+                <nav>
+        <div>
+            <ul>
+                <li><a href="../homepage/homepage.html">HOME</a></li>
+                <li><a href="#">ABOUT</a></li>
+                <li><a href="#">BLOG</a></li>
+                <li><a href="#">CONTACT</a></li> 
+                <li><a href="../submission-page/Submission-page.html">SUBMISSION</a></li>
```
Comment (Olawumi11) [inline homepage/homepage.html]: I think we should name this "News Submission". "Submission" alone seems vague. What do you think?

**Review by Olawumi11 — APPROVED** [review]
Great work. Thanks for taking up the task.

File: homepage/homepage.html (hunk 1 above)
Comment (Egwolo2) [reply homepage/homepage.html]: Well done Everyone.
I agree with Micheal, i think all pages should have uniformity.

**Review by Egwolo2 — APPROVED** [review]

**Review by tundeojediran — APPROVED** [review]

**Pull request merged**